shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-plugin/shardingsphere-sharding-cosid/src/main/java/org/apache/shardingsphere/sharding/cosid/algorithm/keygen/CosIdKeyGenerateAlgorithm.java

```diff
@@ -46,7 +46,7 @@ public final class CosIdKeyGenerateAlgorithm implements KeyGenerateAlgorithm {
     public void init(final Properties props) {
         this.props = props;
         lazyIdGenerator = new LazyIdGenerator(props.getProperty(CosIdAlgorithmConstants.ID_NAME_KEY, IdGeneratorProvider.SHARE));
-        asString = Boolean.parseBoolean(props.getProperty(AS_STRING_KEY, Boolean.FALSE.toString()));
```

Comment:
It has already converted in `TypedSPIRegistry.convertToStringTypedProperties()`.
It is better to use props.getProperty to get the value here.

Reply:
Ok! I found it. 
However, `Properties` does not have such a convention that the value content must be set to a string type.
So developers should be informed of such a convention in the `SPIPostProcessor.init` method comment.